### DISCUSSION TITLE
How to use a repository for retrieval and search data

### DISCUSSION THREAD

**Discussion by hansbak** [romanjrdykyj/select2dot1#13] [Q&A] [answered]
I would like to know how to use this package with a repository using [dio](https://pub.dev/packages/dio)

so when using the search it uses the repository.

can select2dot1 do this?

**Comment by romanjrdykyj** [comment, marked as answer]
No select2dot1 can't do this now, but this feature will be implemented soon! Stay tuned!

**Comment by hansbak** [comment]
Thanks for your quick reply!Regards, HansOn 19 Jan 2023 16:26, Roman Dykyj ***@***.***> wrote:
No select2dot1 can't do this now, but this feature will be implemented soon! Stay tuned!
